browser/performance.ts

```diff
@@ -0,0 +1 @@
+export default performance;
```

Comment:
Ignore this comment if it's a known issue with this Draft PR.

I think a level of indirection is needed here - via `typeof` or `function`. Although this would work in a recent browser, referencing the global  `performance` at top level causes the browser bundle and its tests to fail on node <= v14:

```
$ node-v10.4.1 -e performance 2>/dev/null && echo OK || echo FAIL
FAIL
$ node-v12.13.0 -e performance 2>/dev/null && echo OK || echo FAIL
FAIL
$ node-v14.16.0 -e performance 2>/dev/null && echo OK || echo FAIL
FAIL
$ node-v16.1.0 -e performance 2>/dev/null && echo OK || echo FAIL
OK
```

Reply:
yeah, ideally the global performance object would be used for both browser and node.js. I was under the impression it was supported as a global in v14, but you are right, it's only available in v16, unless someone would do a backport  😞 

if I understood the bundle process correctly, the file used for the node.js build would be this one: https://github.com/rollup/rollup/pull/4340/files#diff-56aca2bfe43f23ab215af1f0c17d5ac2abbed4deb560487408bc588c47c2fc44R1  being replaced with this call: https://github.com/rollup/rollup/blob/da3bb43003e48fbb9815d9bd5aefee55d776b386/rollup.config.ts#L160

originally I wrapped and exported the `now` function only, but I was thinking it's probably better to make the entire `performance` object (for future use) available. I was also thinking to remove the timer Map, and use `performance.measure` and/or `performance observer` instead. but it seems currently it wouldn't be possible to attach any other metadata, e.g. for the value for the memory consumption. (`detail` param was added in v16)

Reply:
ah, I see. did you mean this maybe: https://github.com/rollup/rollup/runs/4723627853?check_suite_focus=true#step:7:5615  just noticed. I have to check it out. I suppose those tests are not running in any browser but in node.js  🤔

Reply:
Yeah, the browser bundle name is a bit of a misnomer.  It really means JS platform independent.